corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -17,11 +17,69 @@ hqDefine("geospatial/js/case_grouping_map",[
         self.coordinates = coordiantes;
         self.caseLink = caseLink;
 
+        // TODO: Group ID needs to be set
+        self.groupId = null;
+
         return self;
     }
 
+    function exportModel() {
+        var self = {};
+
+        self.casesToExport = ko.observableArray([]);
+
+        self.handleExportCSV = function () {
+            const casesToExport = _.map(self.casesToExport(), function (caseItem) {
+                const coordinates = (caseItem.coordinates) ? `${caseItem.coordinates.lng} ${caseItem.coordinates.lat}` : "";
+                return {
+                    'groupId': caseItem.groupId,
+                    'caseId': caseItem.caseId,
+                    'coordinates': coordinates,
+                };
+            });
+            exportToCsv(casesToExport);
+        };
+
+        // If list of propertiesToInclude is not given, will export all properties of objects in itemsArr
```

Comment:
where is this passed, `propertiesToInclude`?

Reply:
Ah, this is an old comment from my original approach where we could specify what object properties we want to export. I have since changed this to simply export all the objec'ts properties.

Will remove this comment.

Reply:
[444424c](https://github.com/dimagi/commcare-hq/pull/33610/commits/444424c741d3f0a11260d8bab39630c063e68c64)